change/@fluentui-eslint-plugin-f8463987-f3be-4b03-a0c2-3420717d4cce.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "Add eslint-plugin-jsdoc to check for jsdoc errors",
+  "packageName": "@fluentui/eslint-plugin",
+  "email": "behowell@microsoft.com",
+  "dependentChangeType": "none"
```

Comment:
Note I changed this to `none`, per https://github.com/microsoft/fluentui/pull/19169/files#r681520686

Reply:
great catch @behowell !

@ecraig12345 do we need to publish this package at all? who consumes this outside our org ? 
if we make it private then we can mitigate such a chore that happened last time with releases.

WDYT?

Reply:
IIRC I originally made it public because some of the custom rules ([outlined in the readme](https://www.npmjs.com/package/@fluentui/eslint-plugin)) might be useful to others. But I have no way to know if anybody else has actually found and used them.